### PR TITLE
Fix broken link to lagom framework

### PR DESCRIPTION
**Pull request by balteo**

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @balteo,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by raboof** [comment]
OK TO TEST

**Comment by raboof** [comment]
Refs #25843

**Review by raboof — APPROVED** [review]
Thanks!

File: akka-docs/src/main/paradox/persistence.md [hunk 1]
```diff
@@ -727,7 +727,7 @@ active entity (`PersistentActor`) for each id within a data center. Lightbend's
 [Multi-DC Persistence](https://developer.lightbend.com/docs/akka-commercial-addons/current/persistence-dc/index.html)
 supports active-active persistent entities across data centers.
 
-The [Lagom framework](https://www.lagom-framework.com), which is built on top of Akka encodes many of the best practices 
+The [Lagom framework](https://www.lagomframework.com), which is built on top of Akka encodes many of the best practices
```
Comment (patriknw) [inline akka-docs/src/main/paradox/persistence.md]: there is one more in persistence-query.md

**Pull request merged**